pkg/Library/CoreFramework/App.js

```diff
@@ -47,7 +47,6 @@ export const createLayer = async (graph, atomEmitter, Composer, Services, name='
 };
 
 export const obliterateLayer = layer => {
-  //globalThis.layers.remove(layer);
   // dispose Atoms
```

Comment:
now that `layer` related methods are in Flan, should App still have the `createLayer` / `obliterateLayer` methods?

Reply:
It's good for Flan to delegate to something stupider, in general.